### ISSUE TITLE
Types not available for css prop with @jsxImportSource theme-ui

### ISSUE THREAD

**Issue by yunyu** [system-ui/theme-ui#1833]
**Describe the bug**
When I try using the `css=` prop in a file with jsxImportSource theme-ui set:

```tsx
/** @jsxImportSource theme-ui */
<div css={[ACTIVE_STYLE, ACTIVE_BADGE_STYLE]} />;
```

I get:

```
  Property 'css' does not exist on type 'ClassAttributes<HTMLDivElement> & HTMLAttributes<HTMLDivElement> & SxProp'.ts(2322)
```

This is mentioned as a valid use case in the docs: https://theme-ui.com/sx-prop#raw-css

**To Reproduce**
1. Create a new file with `/** @jsxImportSource theme-ui */`
2. Create a component with ```const TestComponent = () => <div css={css`display: block;`} />;```
3. TypeScript fails the type check

**Expected behavior**
`css=` should have a type definition present and typed to Emotion's css function output

**Additional context**
This was possible on version 0.3.2, and appears to still be supported in the docs. I am using theme-ui 0.9.1

**Comment by hasparus** [comment]
Thanks for the issue, @yunyu!

I made a PR with a fix already, but until it's released you should be able to work around this in your codebase by adding `css` to `SxProp`.

```ts
// your-code.ts

import { Interpolation } from '@emotion/react'

declare module "@theme-ui/core" {
  export interface SxProp {
    css?: Interpolation<Theme>;
  }
}
```

Here's a CodeSandbox with the snippet above https://codesandbox.io/s/theme-ui-css-workaround-55sf0

**Comment by hasparus** [comment]
:rocket: Issue was released in [`v0.11.0-develop.1`](https://github.com/system-ui/theme-ui/releases/tag/v0.11.0-develop.1) :rocket:

**Comment by hasparus** [comment]
:rocket: Issue was released in [`v0.11.0`](https://github.com/system-ui/theme-ui/releases/tag/v0.11.0) :rocket:
